test/jdk/java/awt/Graphics2D/CopyAreaOOB.java

```diff
@@ -50,73 +106,50 @@ public void paint(Graphics g) {
         g2d.fillRect(0, 0, w, 10);
 
         g2d.setColor(Color.red);
-        g2d.fillRect(0, 10, 50, h-10);
+        g2d.fillRect(0, 10, 50, h - 10);
 
         // copy the region such that part of it goes below the bottom of the
         // destination surface
-        g2d.copyArea(0, 10, 50, h-10, 60, 10);
-
-        Toolkit.getDefaultToolkit().sync();
-
-        BufferedImage capture = null;
-        try {
-            Thread.sleep(500);
-            if (robot == null) robot = new Robot();
-            Point pt1 = getLocationOnScreen();
-            Rectangle rect = new Rectangle(pt1.x, pt1.y, 400, 400);
-            capture = robot.createScreenCapture(rect);
-        } catch (Exception e) {
-            throw new RuntimeException("Problems handling Robot");
-        }
-        // Test pixels
-        testRegion(capture, "green",          0,   0, 400,  10, 0xff00ff00);
-        testRegion(capture, "original red",   0,  10,  50, 400, 0xffff0000);
-        testRegion(capture, "background",    50,  10,  60, 400, 0xff000000);
-        testRegion(capture, "in-between",    60,  10, 110,  20, 0xff000000);
-        testRegion(capture, "copied red",    60,  20, 110, 400, 0xffff0000);
-        testRegion(capture, "background",   110,  10, 400, 400, 0xff000000);
+        g2d.copyArea(0, 10, 50, h - 10, 60, 10);
+        g2d.dispose();
```

Comment:
Why do you disposed of `g2d`? It's a parameter that's passed to you as argument to the `paint` method.